plugin.yaml

```diff
@@ -147,6 +146,21 @@ node_types:
             cloudstack_config:
                 default: {}
 
+    cloudify.cloudstack.nodes.Volume:
+            derived_from: cloudify.nodes.Volume
+            properties:
+                volume: {}
+                use_external_resource:
```

Comment:
what about this then? if we dont currently support using external volumes in cloudstack than this property should not be here. right?